kafka/10broker-config.yml

```diff
@@ -57,6 +57,7 @@ data:
     num.partitions=1
 
     default.replication.factor=3
+    offsets.topic.replication.factor=3
```

Comment:
This property is set to 1 below at line 117.

Reply:
Ah, explains why we didn't get the documented default.